.github/conda_skip.yml

```diff
@@ -0,0 +1,184 @@
+# TODO What do we do with these?
+# I think we can require Docker,modules,Shard 1..Singularity,subworkflows,3 and skip the condas now
+exclude:
+  - path: modules/nf-core/nf-test
+  - profile: conda
+    path: modules/nf-core/angsd/gl
+  - profile: conda
+    path: modules/nf-core/annotsv/installannotations
+  - profile: conda
+    path: modules/nf-core/happy/sompy
```

Comment:
Why is this out of order?

Reply:
I tried to just copy it from the test.yml 🤷🏻‍♂️ Maybe I messed that up, maybe it was just out of order.